LibraBFT/Impl/Consensus/SafetyRules/PersistentSafetyStorage.agda

```diff
@@ -39,3 +27,14 @@ new author waypoint sk = mkPersistentSafetyStorage
      {-, _pssAuthor    =-} author
      {-, _pssWaypoint  =-} waypoint
      {-, _pssObmSK     =-} (just sk)
+
+consensusKeyForVersion : PersistentSafetyStorage → PK → Either ErrLog SK
+consensusKeyForVersion self pk =
+  -- LBFT-OBM-DIFF
+  maybeS (self ^∙ pssObmSK) (Left fakeErr {-"pssObmSK Nothing"-}) $ λ sk →
+    if TODO.makePK sk /= pk
+    then Left fakeErr -- ["sk /= pk"]
```

Comment:
Even though it's just a comment and is consistent with the Haskell code, I can't help saying that `sk /= pk` is a weird error message.  Something in the back of my mind is bothering about that :smile:.

Reply:
The full Haskell code is: 
    `Left (ErrL (here ["sk /= pk"]))`
So the full comment will say 
`PersistentSafetyStorage; consensusKeyForVersion; sk /= pk`